data/projects/semesters/f21-w22/mada.js

```diff
@@ -0,0 +1,66 @@
+const name = 'MADA Project';
+const id = 'mada';
+const detail = 'A project to help MADA manage their volunteers and meal delivery schedule';
+const coverImagePath = '/static/images/projects/mada/mada-problem.jpg';
+const clientDetail =
+  "MADA is a volunteer-based organization that evolved from a small neighbourhood centre to Montreal's central address for fighting poverty in our community. MADA’s MEALS À PARTAGER program helps the disabled and elderly by delivering them fresh homemade meals every day of the week.";
+const clientLink = 'https://madacenter.com/';
+const previousLink = null;
+const clientFeatures = null;
+const problem = 'MADA needs a more efficient method of communicating with volunteers what meals to deliver and when.';
+const problemImagePath = '/static/images/projects/mada/mada-problem.jpg';
+const solution = "We're working with MADA to create a mobile application to communicate each volunteers availability and organize meal delivery.";
```

Comment:
Could we say this instead?

We are working with MADA to create a CRM tool that allows MADA employees to monitor volunteers' availability and organize meal delivery.